schema/va-spec/aac-2017/diagnostic-study-statement-profile-source.yaml

```diff
@@ -20,11 +20,79 @@ $defs:
+            for the assessed variant, based on the curation and reporting conventions of the AMP/ASCO/CAP (AAC) 2017
+            Guidelines. Note that the enumerated value set here is bound to the ‘code’ field of the Coding object that
+            is nested inside a MappableConcept's primary coding. Conditional requirement: Strength is evaluated relative to
+            the direction indicated by the 'direction' attribute. If direction is either 'supports' or 'disputes', the
+            'strength' attribute is required.  If the direction is 'neutral', then the 'strength' attribute is not allowed.
+          properties:
+            primaryCoding:
+              allOf:
+                - $ref: "/ga4gh/schema/gks-core/1.0.0/json/Coding"
+                - oneOf:
+                  - properties:
+                        code:
+                          enum:
+                            - Level A
+                            - Level B
+                            - Level C
+                            - Level D
+                        system:
+                          const: AMP/ASCO/CAP (AAC) Guidelines, 2017
+                  - properties:
+                        code:
+                          enum:
+                            - Level E
+                        system:
+                          const: CIViC SOP AAC Guidelines, 2019
+        classification:
+          description: >-
+            A single term or phrase classifying the subject variant based on the outcome of direction and strength assessments of
+            the Statement's Proposition - reported here using terms from the AMP/ASCO/CAP (AAC) 2017 Guidelines. Note that the
+            enumerated value set here is bound to the ‘code’ field of the Coding object that is nested inside a MappableConcept's
+            primary coding.
+          properties:
+            primaryCoding:
+              allOf:
+                - $ref: "/ga4gh/schema/gks-core/1.0.0/json/Coding"
+                - oneOf:
+                  - properties:
+                        code:
+                          enum:
+                            - Tier I
+                            - Tier II
+                            - Tier III
+                            - Tier IV
+                        system:
+                          const: AMP/ASCO/CAP (AAC) Guidelines, 2017
+                  - properties:
+                        code:
+                          enum:
+                            - Tier I - Sensitivity/Response
+                            - Tier I - Resistance
+                            - Tier I - Adverse Response
+                            - Tier I - Reduced Response
+                            - Tier II - Sensitivity/Response
+                            - Tier II - Resistance
+                            - Tier II - Adverse Response
+                            - Tier II - Reduced Response
```

Comment:
@mbrush @ahwagner @larrybabb So we're using these same values in prognostic, diagnostic, AND therapeutic... I'm not sure if we want that?

Reply:
I agree; and further think that we don't need implementation / resource-specific codes here. This is where we might include extensions that are decided by SC-VCEPs.